### ISSUE TITLE
FineTuning

### ISSUE THREAD

**Issue by wonlee2019** [qingshi9974/ICLR2024-FTIC#1]
Thanks for your contributions!

I have one question:

How can I train the low bitrate model by using well-trained high bitrate model?

How can I train the model optimized with MS-SSIM from the well-trained model optimized with MSE?

Thanks again!

**Comment by qingshi9974** [comment]
You can load the checkpoint of  the pre-trained high bitrate/MSE-optimized model and use a lower learning rate (i.e., 1e-5) for fine-tuning to obtain a low bitrate/MS-SSIM-optimized model.

**Comment by qingshi9974** [comment]
> Thanks for your contributions!
> 
> I have one question:
> 
> How can I train the low bitrate model by using well-trained high bitrate model?
> 
> How can I train the model optimized with MS-SSIM from the well-trained model optimized with MSE?
> 
> Thanks again!

You can load the checkpoint of the pre-trained high bitrate/MSE-optimized model and use a lower learning rate (i.e., 1e-5) for fine-tuning to obtain a low bitrate/MS-SSIM-optimized model.

**Comment by wonlee2019** [comment]
Thanks for your response. I changed the learning rate for training, but it doesn't work.

As same as before, running the code followed by the increase of the GPU-MEM, and it is terminated at 2k MiB (lower than the target 8k MiB).

It's weird. Whatever, thanks again.
